NCS.DSS.GDPRUpdateCustomerInformation/NCS.DSS.GDPRUtility.csproj

```diff
@@ -7,6 +7,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="1.13.1" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.23.0" />
```

Comment:
Uninstall this package - it's not used

Reply:
Pretty sure `Microsoft.Azure.Functions.Worker.Extensions.Timer` provides the timer functionality, not `Microsoft.Azure.Functions.Worker` - try to uninstall it and rebuilt - it should still succeed

Reply:
It's not required: 
![image](https://github.com/user-attachments/assets/4430ea0a-d143-47ba-bef1-7da591f25bbd)

This was deployed to AT without that package - please uninstall